### ISSUE TITLE
phpstan qa failure since last release

### ISSUE THREAD

**Issue by lulhum** [composer/docker#183]
My circeci phpstan qa stopped functioning with last composer release (2.1.1). Tagging an older composer release resolve the issue.

I don't really know what happen, but I have the following output:
```
$ ./vendor/bin/phpstan analyze --level=5 --memory-limit=-1 --debug src

Note: Using configuration file /root/project/phpstan.neon.
Invalid configuration:
Service '0282': Class or interface 'PHPStan\Type\Doctrine\DescriptorRegistry' not found.

Exited with code exit status 1
```

**Comment by ferrastas** [comment]
I guess that you mean `2.1.0` as the image for `2.1.1` does not exist (and most likely fixes what you are reporting), check https://github.com/composer/composer/pull/9935

**Comment by alcohol** [comment]
Probably fixed with `2.1.1`, otherwise wait for `2.1.2` to  be merged into official-images upstream.
